### PR TITLE
Updating some grammar and example layouts

### PR DESCRIPTION
**Pull request by cdietschrun**
#1413 simple grammar tweak
#1415 reordering of the examples to by consistent

### CODE REVIEW

**Comment by PRMerger15** [comment]
@cdietschrun : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger4** [comment]
@cdietschrun : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@cdietschrun : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger5** [comment]
@cdietschrun : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger-2** [comment]
@cdietschrun : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@cdietschrun : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by ktoliver** [comment]
@tommoser Could you review the proposed code changes in **mip/develop/concept-delegation.md**?

All other changes look good to me. . .

Thanks.

**Comment by batamig** [comment]
Hi @cdietschrun - thanks again for contributing to docs! 

I'm tagging the focal points on the team and have closed the related issues so as to consolidate any conversation.
I also agree that the grammar fixes look right, but we need approval before merging the code changes.

@tommoser  and @msmbaldwin, can you take a look?

File: mip/develop/concept-delegation.md
Comment (tommoser) [inline mip/develop/concept-delegation.md:126]: I think these are mixed up. The example headers were switched and said consumption for publishing and vice versa. This looks like you updated the sample code and then reversed the headers. We can either switch the headers only or leave the headers and take the sample code updates.

**Review by tommoser — CHANGES_REQUESTED** [review]
Thanks for the PR and feedback! It looks good other than that we need to clarify the change around the delegation scenarios and the consumption/publishing examples.

**Comment by batamig** [comment]
Hi @tommoser - thanks for the feedback!
@cdietschrun, can you fix the confusion in the example headers before we merge this PR?
@tommoser, once that item is fixed, if there are other changes required, let's take offline into the backlog to update as needed.

**Pull request closed without merging**
